.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Packages
+
+on:
+  workflow_call:
```

Comment:
this trigger is not needed on this workflow

Reply:
removed